### PR TITLE
[build] Ensure /usr/lib/systemd/system/ directory exists before referencing

### PR DESCRIPTION
**Pull request by abdosi**
**- Why I did it**
Fix the Build Failure on 201911 (Stretch) because of  directory /usr/lib/systemd/system/ does not exist.creating manually the directory as part of build image. 
Change should not harm Master (buster) where the directory is created by Linux

**- How I did it**
using mkdir -p

**- How to verify it**
Verified 201911 Build is fine after the change

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -40,6 +40,10 @@ FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES="$FILESYSTEM_ROOT_USR_SHARE_SONIC/temp
 FILESYSTEM_ROOT_ETC="$FILESYSTEM_ROOT/etc"
 FILESYSTEM_ROOT_ETC_SONIC="$FILESYSTEM_ROOT_ETC/sonic"
 
+# Create the directoy. This is needed for Stretch and might not need
```
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2:43]: s/directoy/directory/

File: files/build_templates/sonic_debian_extension.j2 [hunk 2]
```diff
@@ -40,6 +40,10 @@ FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES="$FILESYSTEM_ROOT_USR_SHARE_SONIC/temp
...
+# be needed for Buster. It should not harm anyways
+sudo mkdir -p $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
+
```
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2:46]: Suggest to move this just below where all other system directories are created (lines 79-83)

**Comment by jleveque** [comment]
@abdosi: FYI, I updated the PR title.

File: files/build_templates/sonic_debian_extension.j2 (hunk 2 above)
Comment (abdosi) [reply files/build_templates/sonic_debian_extension.j2]: @jleveque Done

File: files/build_templates/sonic_debian_extension.j2 (hunk 1 above)
Comment (abdosi) [reply files/build_templates/sonic_debian_extension.j2]: @jleveque Done

File: files/build_templates/sonic_debian_extension.j2 [hunk 3]
```diff
@@ -40,6 +40,7 @@ FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES="$FILESYSTEM_ROOT_USR_SHARE_SONIC/temp
 FILESYSTEM_ROOT_ETC="$FILESYSTEM_ROOT/etc"
 FILESYSTEM_ROOT_ETC_SONIC="$FILESYSTEM_ROOT_ETC/sonic"
 
+
```
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2:43]: Delete this extra blank line.

Comment (abdosi) [reply files/build_templates/sonic_debian_extension.j2]: @jleveque Just did it before you comment.

**Pull request merged**